website/docs/quickstart_admin.md

```diff
@@ -0,0 +1,87 @@
+---
+id: quickstart_admin
+title: JBrowse 2 admin quickstart
+toplevel: true
+---
+
+This guide will introduce several new administration features provided with JBrowse 2.
+
+### Pre-requisites
+
+This tutorial requires having the following software installed
+
+- [JBrowse 2 CLI tools](quickstart_web#install-the-cli-tools)
+
+- [JBrowse 2 web application](quickstart_web#using-jbrowse-create-to-install-jbrowse)
+
+### Starting JBrowse2 Admin Server
```

Comment:
this is fairly picky but can we make subheadings use sentence-casing, e.g.  ```### Starting JBrowse2 admin server```

Reply:
Sure, it should be consistent with other guides